modulefiles/mesoscale/mesoscale_stats.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # modulefile for EVS mesoscale component
 
-set -x
+set +x
```

Comment:
@BinbinZhou-NOAA Please change this back to "set -x". Thanks!

Reply:
Right ... this will get fixed by simply making sure the modfulefiles in this PR match develop. The modulefiles for mesoscale in develop are correct.